handlers/orderHandler.js

```diff
@@ -0,0 +1,33 @@
+const logger = require('../library/Logging');
+const Order = require('../model/orderModel');
+
+const allOrders = async (req,res) => {
+	const orderLists = await Order.find();
+
+	if(!orderLists) {
+		res.status(500).json({error:logger.error('Internal server error order lidt cant be found'),
+		success:false
+		})
+	};
```

Comment:
```suggestion

	
```
Please don't do this, the code will never run and it doesn't make sense to return an internal server error when you simply don't have items in a collection, please relate your reason for this